ui/app/(ai)/analyst/utils.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  AIMessage,
+  BaseMessage,
+  ChatMessage,
+  HumanMessage,
+} from "@langchain/core/messages";
+import type { Message } from "ai";
+
+// https://stackoverflow.com/questions/79081298/how-to-stream-langchain-langgraphs-final-generation
+/**
+ * Converts a Vercel message to a LangChain message.
+ * @param message - The message to convert.
+ * @returns The converted LangChain message.
+ */
+export const convertVercelMessageToLangChainMessage = (
```

Comment:
Following up on this comment: https://github.com/prowler-cloud/prowler/pull/7508#discussion_r2052086824 

Could we please use the `/app` folder only for routing purposes?

Thank you!

Reply:
Moved code to `/lib/lighthouse/utils.ts`